src/WebUI/Components/ServicesComponent.tsx

```diff
@@ -142,7 +136,8 @@ export class ServicesComponent extends BaseComponent<IServicesComponentPropertie
             minWidth: 110,
             maxWidth: 110,
             headerClassName: headerColumnClassName,
-            columnActionsMode: ColumnActionsMode.disabled
+            columnActionsMode: ColumnActionsMode.disabled,
+            className: css(columnContentClassName, "sc-wide-col")

```

Comment:
sc-wide-col, this is no use I believe, make the min and max to 140, it is fine, to have a little bigger column